test.kt

```diff
@@ -11,9 +11,7 @@ fun myFunction() {
     println("line 11")
     println("line 12")
     println("line 13")
-    println("line 14")
-    println("line 15")
-    println("line 16")
+    println("line 14") // Random change
     println("line 17")
     println("line 18")
     println("line 19")
```

Comment:
Multi-line comment starting on removed line 14, ending on 17 (side == RIGHT)
```suggestion
line_1
line_2
line_3
```